requirements.txt

```diff
@@ -1 +1,3 @@
+pathlib2>=2.3.2
 pytest>=2.8.6
+scandir>=1.8
```

Comment:
I would prefer to leave this file as is, as it contains the requirements needed for `pyfakefs`, and put the extra requirements into an extra requirements file.